docs/development/vsphere-dev.md

```diff
@@ -73,21 +73,43 @@ vSphere cloud provider support in kops is a work in progress. To try out deployi
 ### Pre-requisites
 + vSphere with at least one ESX, having sufficient free disk space on attached datastore. ESX VM's should have internet connectivity.
 + Setup DNS following steps given in relevant Section above.
-+ Create the VM using this template (TBD).
++ Upload VM template. Steps:
+1. Login to vSphere Client.
+2. Right-Click on ESX host on which you want to deploy the template.
+3. Select Deploy OVF template.
+4. Copy and paste URL for [OVA](TBD).
```

Comment:
Was waiting for @prashima to upload it.
I will try to upload a version to google storage.